src/data/connectors.js

```diff
@@ -2,57 +2,67 @@ import Sequelize from 'sequelize';
 
 import { get as httpsGet } from 'https';
 
-import dotenv from 'dotenv';
+import * as dotenv from 'dotenv';
+
+import db from './db/models';
 
 dotenv.config();
 
 export class IrsDbConnector {
-
   constructor() {
-    this.db = new Sequelize(process.env.PG_DATABASE, process.env.PG_USER, process.env.PG_PASSWORD, {
-      host: process.env.PG_HOST,
-      port: process.env.PG_PORT,
-      dialect: 'postgres',
-      dialectOptions: {
-        ssl: (process.env.PG_SSLMODE === 'require'),
+    this.db = new Sequelize(
+      process.env.PG_DATABASE,
+      process.env.PG_USER,
+      process.env.PG_PASSWORD,
+      {
+        host: process.env.PG_HOST,
+        port: process.env.PG_PORT,
+        dialect: 'postgres',
+        dialectOptions: {
+          ssl: process.env.PG_SSLMODE === 'require',
+        },
+      }
+    );
+
+    this.IrsDbModel = this.db.define(
+      'irsdb',
```

Comment:
Would this make more sense in schema.js or in a `modelsi/irs.js` file?

Reply:
yes; i'm working on this now actually!